meta/runtime.yml

```diff
@@ -1,2 +1,9 @@
 ---
 requires_ansible: '>=2.9'
+
+action_groups:
+  k8s:
+    - k8s
+    - k8s_auth
```

Comment:
I don't think we'd need this in this collection—users with this collection would only be using `openshift_auth`.

Reply:
I was sure how to play that with the aliasing. I can remove it.

Reply:
```suggestion
```